src/util/logger/Logger.cpp

```diff
@@ -32,8 +33,11 @@ void Logger::log(std::string message, const std::string log_type) {
     } else if (log_type.compare("error") == 0) {
         daily_logger->error(message);
         logger->error(message);
+    } else if (log_type.compare("debug") == 0) {
+        daily_logger->debug(message);
+        logger->debug(message);
     } else {
-        //std::cout << "Invalid logging type" << std::endl;
+        // std::cout << "Invalid logging type" << std::endl;
```

Comment:
Lets remove this commented line.

Reply:
Fixed in https://github.com/miyurud/jasminegraph/pull/115/commits/ed31ba0af33e9629173808960eef3cfe90c5dbe9

Reply:
Done